ecc/bls12-377/g1.go

```diff
@@ -421,8 +428,15 @@ func (p *G1Jac) mulWindowed(a *G1Jac, s *big.Int) *G1Jac {
 	var res G1Jac
 	var ops [3]G1Jac
 
-	res.Set(&g1Infinity)
 	ops[0].Set(a)
+	e := s
```

Comment:
@yelhousni @ThomasPiellard  this snippet is weird?  didn't review this PR but to me this code is useless.
What does this code do; assign a *big.Int, overwrites it with an object from a pool, and Neg(s) into e, the object of the pool before discarding it.